### ISSUE TITLE
Hacktoberfest Contribute

### ISSUE THREAD

**Issue by prathamesh-borse** [prathamesh-borse/Hacktoberfest_2021#1]
Feel free to add everything you want from memes to blogs to codes to even show off your projects!

Create a new issue that You want to contribute to this repository and Make a PR from a different branch.

**Comment by Sid672** [comment]
I created a pull request please review it. https://github.com/prathamesh-borse/Hacktoberfest_2021/pull/5

**Comment by juhiechandra** [comment]
Can we add our projects build on mern stack ?

**Comment by prathamesh-borse** [comment]
> I created a pull request please review it. #5

Reviewed It and Merged It into the Main🤞☺️

**Comment by prathamesh-borse** [comment]
> Can we add our projects built on mern stack ?

You can add Whatever You feel Like whether it's project code or anything related to tech, Just Go-ahead And Make a PR.

**Comment by tausiq2003** [comment]
hey there can i work on this for hacktoberfest 2021 and which type of code should I write

**Comment by prathamesh-borse** [comment]
@tausiq2003 Which Code You Want to Write?

**Comment by tausiq2003** [comment]
Merging two linked lists

**Comment by tausiq2003** [comment]
ok thanks.

**Comment by tausiq2003** [comment]
bro please add this tag hacktoberfest

**Comment by tausiq2003** [comment]
you have added Hacktoberfest

**Comment by prathamesh-borse** [comment]
> you have added Hacktoberfest

Yes.

**Comment by tausiq2003** [comment]
wait

**Comment by tausiq2003** [comment]
![Screenshot 2021-09-30 211033](https://user-images.githubusercontent.com/80088029/135487308-c3ecc366-d746-4e76-820d-e9bc7826a98d.jpg)

**Comment by tausiq2003** [comment]
see the red line

**Comment by tausiq2003** [comment]
H should be h

**Comment by prathamesh-borse** [comment]
@tausiq2003 are u here For Doing Contribution or Finding Typos in Labels.
If you really want to Contribute then make a fork and make changes don't spam on issue section.!

**Comment by tausiq2003** [comment]
I will delete all comments but you have to add a label hacktoberfest because it is necessary and it is rule don't get irritated.

**Comment by tausiq2003** [comment]
see on the rules section

**Comment by prathamesh-borse** [comment]
I have already Added Please Check Again ☺️

**Comment by prathamesh-borse** [comment]
> I will delete all comments but you have to add a label hacktoberfest because it is necessary and it is rule don't get irritated.

Already Updated The Label Please Check ✅

**Comment by tausiq2003** [comment]
'hacktoberfest' the particular label bro.

**Comment by Sahaj-Srivastava24** [comment]
Hey, looking for PRs for hacktober.. wondering what I can code done.. any suggestions..?

**Comment by prathamesh-borse** [comment]
@tausiq2003 So, What do You want Man?

**Comment by prathamesh-borse** [comment]
> Hey, looking for PRs for hacktober.. wondering what I can code done.. any suggestions..?

What Technologies Have You Known About?

**Comment by Sahaj-Srivastava24** [comment]
js, node, py..
Also, was there an issue with the PR i did..? I guess I made it in another directory

**Comment by prathamesh-borse** [comment]
> js, node, py.. Also, was there an issue with the PR i did..? I guess I made it in another directory

Merged Now Man🤞👍

**Comment by prathamesh-borse** [comment]
> ok can i work on this for now

Where You Have to Work?

**Comment by Sahaj-Srivastava24** [comment]
Im guessing the hacktober servers a down.. none of my PRs are getting counted.. weird
Anyways, ill come back if this one counts.. Kudos till then

**Comment by 123mansha** [comment]
Can I add a Leetcode Question Solution?

**Comment by AkshaiBaruah** [comment]
Can I add a simple beginner memes app written in kotlin so that beginners can further improve on it?

**Comment by karandeeppotato** [comment]
Hello there, please consider this PR #15

**Comment by Sahaj-Srivastava24** [comment]
@prathamesh-borse boy i got a couple of prs lined up.. it'd be great If you can merge them

**Comment by gauri-1312** [comment]
Hey , made a pull request . Please review it

**Comment by prathamesh-borse** [comment]
> Hey , made a pull request . Please review it #29

Sure. 💯

**Comment by prathamesh-borse** [comment]
> Can I add a Leetcode Question Solution? @prathamesh-borse

Yes, Go ahead and add it to a leetcode folder.

**Comment by prathamesh-borse** [comment]
> Can I add a simple beginner memes app written in kotlin so that beginners can further improve on it? @prathamesh-borse

Yes, Go Ahead create a new issue and make a PR with a different branch.

**Comment by prathamesh-borse** [comment]
> Hello there, please consider this PR #15 @prathamesh-borse

Its already Merged into the main branch.

**Comment by joyeta148** [comment]
@prathamesh-borse I have created #32 and #33 PR  please review them.

**Comment by Ayushi7456** [comment]
Hello, I want to add a dice game. should i make new issue for the same?

**Comment by prathamesh-borse** [comment]
> Hello, I want to add a dice game. should i make new issue for the same?

Yes, Create new issue and Start Working On it. 👍

**Comment by mitalsapkale01** [comment]
@prathamesh-borse I'm read out the all those comments it's look like cool. 
I'm also interested for this! Can u suggest me what's I for contribute too?

**Comment by prathamesh-borse** [comment]
> @prathamesh-borse I'm read out the all those comments it's look like cool. I'm also interested for this! Can u suggest me what's I for contribute too?

You Can Add Yourself in Readme, Add Your Favourite Project By Creating New Branch, Add The DSA Problem That You Have Solved.
This Way You Can Contribute.
If You want to Contribute a Project Then Please Create a New Issue and Start Working On it.

**Comment by Subhadiptech** [comment]
Can I add my android project  here?
@prathamesh-borse

**Comment by tausiq2003** [comment]
is this repo included in hacktoberfest or its a excluded project.

**Comment by ShivamChoudhary17** [comment]
Hi!
I would like to add flight fare prediction and heart disease with code and detail explanation,
I also have a project in dart.
If you allow may I add these to repo
Could you please assign issue to me?
Thank You!

**Comment by prathamesh-borse** [comment]
> Hi! I would like to add flight fare prediction and heart disease with code and detail explanation, I also have a project in dart. If you allow may I add these to repo Could you please assign issue to me? Thank You!

Create a New Issue and Goahead.

**Comment by prathamesh-borse** [comment]
> Can I add my android project here? @prathamesh-borse

Yes Create a New Issue and Go Ahead

**Comment by henriquehorbovyi** [comment]
I've created a pull request #97 😺
